config.yaml

```diff
@@ -193,7 +193,7 @@
             format: table
         final_format: json
         custom_transformations:
-          overall_max_score: 5
+          overall_max_score: 7
```

Comment:
Can you briefly explain what this change is doing?

Reply:
The overall scores were previously calculated by adding the genetics, omics, and lit scores together and the max possible overall score was 7. For the upcoming publication, they decided to drop the lit score so now the overall is genetics + omics, with a max possible value of 5. 

We use the configured x_max_score value in transform.py's calculate_distribution method to define the bins for the score distributions.

Reply:
...and we aren't ready for the new 5 point scores yet, which is why I reverted this change. I'm waiting until we get your updates out into dev before I start changing more things.